READMEs/azure-container-deployment.md

```diff
@@ -19,15 +19,28 @@ Follow the instructions from [install pdr-backend](./install.md)
 Set the following environment variables:
 
 - PRIVATE_KEY: Set it to the private key of the wallet you're going to use.
-- PAIR_FILTER: List of pairs to filter (separated by comma), if empty the app will fetch all available pairs. For this example, set it to `"BTC/TUSD"`.
-- TIMEFRAME_FILTER: Timeframes to filter (separated by comma), if empty the app will fetch all available timeframes. For this example, set it to `5m`.
-- SOURCE_FILTER: Price sources filter (separated by comma), if empty the app will fetch all available sources. For this example, set it to `binance`
-- RPC_URL: The RPC URL of the network, set this to Sapphire testnet Websocket RPC URL: `https://testnet.sapphire.oasis.dev`
-- SUBGRAPH_URL: The Ocean subgraph url, set this to Sapphire testnet subgraph URL: `https://v4.subgraph.oasis-sapphire-testnet.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph/graphql`
+- PAIR_FILTER: List of pairs to filter (separated by comma), if empty the app will fetch all available pairs.
+- TIMEFRAME_FILTER: Timeframes to filter (separated by comma), if empty the app will fetch all available timeframes.
+- SOURCE_FILTER: Price sources filter (separated by comma), if empty the app will fetch all available sources.
+- RPC_URL: The RPC URL of the network.
+- SUBGRAPH_URL: The Ocean subgraph url.
```

Comment:
You can delete these lines.

Some envvars are self-documenting. Eg RPC_URL, SUBGRAPH_URL

Some envvars could use a bit of documentation. In that case, do a comment right above the command. (Many other pdr-backend READMEs do this, check it out. (And in some cases, an inline comment will work too.)